modules/oadp-scheduling-backups.adoc

```diff
@@ -59,6 +59,16 @@ spec:
 EOF
 ----
 <1> `cron` expression to schedule the backup, for example, `0 7 * * *` to perform a backup every day at 7:00.
+
+[NOTE]
+====
+To schedule a backup every number of minutes, enter the number of minutes to start the next backup in the following format.
+----
+  schedule: "*/10 * * * *"
```

Comment:
can we have a placeholder like `<duration_in_minutes>` instead of adding a separate statement below?

Reply:
Changed as requested